release/models/bgp/openconfig-bgp-common-multiprotocol.yang

```diff
@@ -185,6 +192,26 @@ submodule openconfig-bgp-common-multiprotocol {
     leaf enabled {
       type boolean;
       default false;
+      description
+        "This leaf indicates whether the AFI-SAFI is
+        enabled for all neighbors or groups";
```

Comment:
```suggestion
        "This leaf indicates whether the AFI-SAFI is enabled for all
        neighbors or groups.  For historical reasons AF_IPV4 must be 
        enabled by default and all other AF are disabled by default.  This
        supports implementations of BGP RFC4271 which only defined
        IPv4 and is always enabled when BGP is configured.";
```

Reply:
LGTM

Reply:
I don't agree with adding this. Please can we leave this clarification for a subsequent CL.

Reply:
changing defaults to be inconsistent based on out of touch and out of date reasons is counter to the goal of openconfig